src/internet_identity/internet_identity.did

```diff
@@ -758,6 +770,8 @@ service : (opt InternetIdentityInit) -> {
     // =========================
     openid_credential_add : (IdentityNumber, JWT, Salt) -> (variant { Ok; Err : OpenIdCredentialAddError });
     openid_credential_remove : (IdentityNumber, OpenIdCredentialKey) -> (variant { Ok; Err : OpenIdCredentialRemoveError });
+    openid_prepare_delegation : (JWT, Salt, SessionKey) -> (variant { Ok : OpenIdPrepareDelegationResponse; Err : OpenIdDelegationError });
```

Comment:
We should also change the ii_spec file to include the new endpoints. But not in this PR, let's do it when we enable the flat for production.